docs/Populate.md

```diff
@@ -0,0 +1,118 @@
+# Populate Exchange
+
+`populate` is an exchange for auto-populating fields in your mutations.
```

Comment:
```suggestion
The `populateExchange` allows you to use the `@populate` directive in your mutations to automatically add selection sets to them based on all of your queries.
This allows your mutations to automatically fetch data based on which fields are actually in use.
If your mutation has a `viewer` field that resolves back to your `Query` type, it will also allow you to automatically refetch data that is used in other parts of your app, giving you back more functionality of `urql`'s default `cacheExchange` with the more capable Graphcache `cacheExchange`.
```

Reply:
I think this information is useful but I'm in the mindset that less is more - IMHO the Urql docs as we have them are already too verbose/in-depth for most users.

> automatically add selection sets

Most users of GraphQL aren't building libraries or working with ASTs. This info is useful for contributors but not end users.

If you and @JoviDeCroock both think this level of detail adds value then lets' go ahead. I think the existing info, along with examples, is concise enough.